iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -814,14 +825,25 @@ IOTHUB_CLIENT_RESULT IoTHubClient_LL_UploadMultipleBlocksToBlob_Impl(IOTHUB_CLIE
 
                                             /*do step 3*/ /*try*/
                                             /*Codes_SRS_IOTHUBCLIENT_LL_02_091: [ If step 2 fails without establishing an HTTP dialogue, then the HTTP message body shall look like: ]*/
-                                            if (BUFFER_build(responseToIoTHub, (const unsigned char*)FILE_UPLOAD_FAILED_BODY, sizeof(FILE_UPLOAD_FAILED_BODY) / sizeof(FILE_UPLOAD_FAILED_BODY[0])) == 0)
+                                            STRING_HANDLE failed_response = STRING_construct_sprintf(FILE_UPLOAD_FAILED_BODY, STRING_c_str(correlationId));
+                                            if(failed_response == NULL)
+                                            {
+                                                LogError("STRING_construct_sprintf failed");
```

Comment:
This message is identical to the one above so we can't really tell which one failed.

Reply:
the LogError expands to put the line number and file where it failed to give greater context.